rskj-core/src/main/java/co/rsk/util/PreflightChecksUtils.java

```diff
@@ -0,0 +1,77 @@
+package co.rsk.util;
+
+import co.rsk.RskContext;
+import co.rsk.config.NodeCliFlags;
+import com.google.common.annotations.VisibleForTesting;
+import org.apache.commons.lang3.StringUtils;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.util.Arrays;
+
+/**
+ * Created by Nazaret García on 21/01/2021
```

Comment:
Add a comment explaining briefly what it does.

Reply:
Done!